Server/src/opcserver_open62541.cpp

```diff
@@ -57,13 +57,20 @@ int OpcServer::start()
         throw_runtime_error_with_origin("UA_Server_new failed");
     m_nodemanager->linkServer(m_server);
     m_nodemanager->afterStartUp();
+    UA_StatusCode status = UA_Server_run_startup(m_server);
```

Comment:
Don't actually know what the call to ``` UA_Server_run_startup``` so this is a rater uninformed statement....

This code just reads rather weirdly: I mean we call a method ```m_nodemanager->afterStartUp();``` then immediately after call a method called ```UA_Server_run_startup(m_server)```. We are talking about different things starting here? Maybe it would help if we named variable ```status``` to give a further hint as to what just started.

Reply:
Yes, the node manager is obviously a different thing than the server, so is there any confusion here really?